### PR TITLE
{math}[foss/2019a] JAGS v4.3.0, rjags v4-9

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f9c88d18662f928b69846bb4d5b91e85 for a full test report.

File: easybuild/easyconfigs/j/JAGS/JAGS-4.3.0-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description = """JAGS is Just Another Gibbs Sampler.  It is a program for analysis 
+ of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [('http://sourceforge.net/projects/mcmc-jags/files/JAGS/%(version_major)s.x/Source/', 'download')]
+sources = [SOURCE_TAR_GZ]
+checksums = ['8ac5dd57982bfd7d5f0ee384499d62f3e0bb35b5f1660feb368545f1186371fc']
+
+configopts = ' --with-blas="-lopenblas" --with-lapack="-lopenblas" '
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/j/JAGS/JAGS-4.3.0-foss-2019a.eb]: why hardcoding `-lopenblas` here? using `$LIBBLAS` and `$LIBLAPACK` is in general better, for instance to allow using `--try-toolchain`

Comment (lexming) [reply easybuild/easyconfigs/j/JAGS/JAGS-4.3.0-foss-2019a.eb]: good suggestion, I had no idea those variables existed. Fixed :+1:

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9e90af80803e05addf4b1bd1e2729b3a for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/8b3089c3b71767fcde3686f815ac0884 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @lexming!

**Pull request merged**